nebullvm/operations/root/black_box/utils.py

```diff
@@ -0,0 +1,128 @@
+import logging
```

Comment:
are you sure that the functions in this module will be used just by the blackbox app?